eng/targetframeworksuffix.props

```diff
@@ -88,4 +87,10 @@
       </PropertyGroup>
     </When>
   </Choose>
+
+  <PropertyGroup>
+    <TargetsUnixOrBrowser Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'">true</TargetsUnixOrBrowser>
+    <TargetsLinuxOrBrowser Condition="'$(TargetsLinux)' == 'true' or '$(TargetsBrowser)' == 'true'">true</TargetsLinuxOrBrowser>
```

Comment:
These don't make much sense to me (especially TargetsLinuxOrBrowser) and don't see reason to introduce them. The commonality between Browser and Unix is just not there

Reply:
We are currently using the unix or linux implementation for  libraries that dont have browser specific code

Reply:
Which is wrong

Reply:
do we want to skip those libraries ? can you provide the libraries which we want to target on browser ?

Reply:
We need every library to provide an impl for browser if it provides a ref on ‘net5’.  This means either targeting browser, net5, or netstandard.

@marek-safar would you prefer we use `'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'` in the projects themselves?  @anipik is trying to make some progress here without taking on too much. We don’t expect this PR to implement libraries across the board for browser. We are trying to make incremental progress towards the goal of providing the right set of configurations. Folks will still need to go through the tests to make things pass by providing better implementations. I think Unix/Linux is a better starting point than windows for things we don’t classify up front as not supported.

Reply:
> would you prefer we use '$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true' in the projects themselves

Yes

> We don’t expect this PR to implement libraries across the board for browser.

That's my expectation as well. @steveisok team is working on that in parallel